.github/ISSUE_TEMPLATE/bug-report.yaml

```
@@ -0,0 +1,45 @@
+---
+name: 🐞 App Bug
+description: Create a report to help us improve Promptmeteo by reproducing and fixing bugs
+title: "[BUG]: "
+labels:
+  - "bug"
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **Thanks :heart: for taking the time to fill out this bug report!**
+        We kindly ask that you search to see if an issue [already exists](https://github.com/paradigmadigital/Promptmeteo/issues?q=is%3Aissue+sort%3Acreated-desc+) for the bug you encountered.
+
+  - type: textarea
+    attributes:
+      label: Current Behavior
+      description: |
+        A clear and concise description of the issue you're experiencing. Include code and the data if possible to reproduce the behaviour.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: |
+        A clear and concise description of what you expected to happen.
+    validations:
+      required: false
+
+  - type: textarea
+    attributes:
+      label: Environment
+      description: |
+        Information about your environment where you see the bug.
```

I find this wording a little unnatural and inconsistent with the rest of the tenses.
```suggestion
        Information about the environment where you saw the bug.
```